GVRf/Framework/framework/src/main/jni/vulkan/vk_render_to_texture.h

```diff
@@ -76,6 +76,7 @@ class VkRenderTexture : public RenderTexture
 
     // Copy data in pixel buffer to client memory. This function is synchronous. When
     // it returns, the pixels have been copied to PBO and then to the client memory.
+    virtual bool readRenderResult(uint8_t *readback_buffer){}
     virtual bool readRenderResult(uint8_t **readback_buffer);
```

Comment:
``bool readRenderResult(uint8_t **readback_buffer)`` doesn't have to be virtual anymore